jax/_src/numpy/lax_numpy.py

```diff
@@ -961,9 +961,73 @@ def matrix_transpose(x: ArrayLike, /) -> Array:
   return lax.transpose(x, axes)
 
 
-@util.implements(np.rot90, lax_description=_ARRAY_VIEW_DOC)
 @partial(jit, static_argnames=('k', 'axes'))
 def rot90(m: ArrayLike, k: int = 1, axes: tuple[int, int] = (0, 1)) -> Array:
+  """Rotate an array by 90 degrees counterclockwise in the plane specified by axes.
+
+  JAX implementation of :func:`numpy.rot90`.
+
+  Args:
+    m: input array. Must have ``m.ndim >= 2``.
+    k: int, optional, default=1. Specifies the number of times the array is rotated.
+    axes: tuple of 2 integers, optional, default= (0, 1). The axes define the plane
+      in which the array is rotated. Both the axes must be different.
+
+  Returns:
+    An array containing the elements of ``m`` rotated by 90 degrees.
+
+  Note:
+    - Unlike :func:`numpy.rot90`, :func:`jax.numpy.rot90` will return a copy rather
```

Comment:
I don't think we need to say this here – it's true of every JAX API.

The reason we included this text in `lax_description` previously was that the numpy docs explicitly mention that the output is a view rather than a copy,. Now that we're not using NumPy's docs, we don't need text that corrects them.

If you're worried about this, you could mention under `Returns:` that the result is a copy of the input

Reply:
Thanks! I have seen a similar note in APIs like `jnp.transpose` and `jnp.matrix_transpose` in the updated docs. So, I have included the note for this API also. Can we remove the note in those APIs also?